ui/src/pages/Information/Npmpackages/components/Package.tsx

```diff
@@ -1,11 +1,11 @@
 import { Card, Col, Skeleton, Space, Tag, Typography } from "antd";
 import React from "react";
-import { Package } from "pages/Information/Npmpackages/utils/types";
+import { Package as PackagType } from "pages/Information/Npmpackages/utils/types";
```

Comment:
??

Reply:
rename it because of name conflict,

component name is Package and another Package is type, so I rename it